tests/performance/density.go

```diff
@@ -0,0 +1,316 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package performance
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+
+	kvv1 "kubevirt.io/client-go/api/v1"
+	cd "kubevirt.io/kubevirt/tests/containerdisk"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/util"
+
+	e2emetrics "kubevirt.io/kubevirt/tests/performance/metrics"
+)
+
+type DensityTest struct {
+	NumVMs               int               `yaml:"num_vms"`                 // number of vms
```

Comment:
```suggestion
	VMCount               int               `yaml:"vm_count"`                 // number of vms
```
I think it's often suggested to use `count` instead of `number` or `numberOf`: https://stackoverflow.com/questions/6358588/how-to-name-a-variable-numitems-or-itemcount

Reply:
I will update to count